providers/openshift.md

```diff
@@ -147,20 +71,20 @@ vm openshift on your development machine) but is an OpenShift v4 cluster.
 
 #### Requirements
 
-* [`CodeReady Containers`](https://github.com/code-ready/crc) ([Installation Instructions](https://code-ready.github.io/crc/#installation_gsg))
-* A compatible [Operating System](https://code-ready.github.io/crc/#minimum-system-requirements-operating-system_gsg) and the [Required software packages](https://code-ready.github.io/crc/#required-software-packages_gsg)
+* [`CodeReady Containers`](https://github.com/crc-org/crc) ([Installation Instructions](https://crc.dev/crc/getting_started/getting_started/installing/))
+* A compatible [Operating System](https://crc.dev/crc/getting_started/getting_started/installing/#_operating_system_requirements)
 
 #### Quickstart
 
-1. [Download](https://cloud.redhat.com/openshift/install/crc/installer-provisioned) ane extract the latest release of CRC
+1. [Download](https://console.redhat.com/openshift/create/local) ane extract the latest release of CRC
 
     ```console
     tar xfJ crc-linux-amd64.tar.xz
     ```
 2. Copy the crc binary to a location on your PATH
 
     ```console
-    sudo cp crc-linux-1.8.0-amd64/crc /usr/local/bin
+    sudo install crc-linux-2.39.0-amd64/crc /usr/local/bin/crc
```

Comment:
Using `install` will work even if crc is currently running (cp fails with a text file busy error)